src/routes/mentions-legales/+page.svelte

```diff
@@ -0,0 +1,38 @@
+<main class="rounded">
+    <h1>Mentions Légales</h1>
+
+    <p>Conformément aux dispositions de la loi n° 2004-575 du 21 juin 2004 pour la confiance en l'économie numérique, il est précisé aux utilisateurs du site mcbookshelf l'identité des différents intervenants dans le cadre de sa réalisation et de son suivi.</p>
+
+    <h2>Edition du site</h2>
+
+    <p>Le présent site, accessible à l’URL mcbookshelf.dev (le « Site »), est édité par :</p>
+    <p>L’association Altearn, enregistrée auprès de la préfecture/sous-préfecture de 75 - Préfecture de police de Paris sous le numéro W061014075, ayant son siège situé à Appartement 561, 16 rue Curial, 75019 PARIS, représentée par Vincent Foriel dûment habilité.</p>
+
+    <h2>Hébergement</h2>
+
+    <p>Le Site est hébergé par la société O2Switch, situé 222 Boulevard Gustave Flaubert, 63000 Clermont-Ferrand, (contact téléphonique : (+33) 4 44 44 60 40).</p>
+
+    <h2>Directeur de publication</h2>
+
+    <p>Le Directeur de la publication du Site est l’association Altearn.</p>
```

Comment:
```suggestion
    <p>La direction de publication du Site est assurée par le·a président·e de l'association Altearn.</p>
```
Le directeur de publication est une personne physique, pas morale